registration/exceptions.py

```diff
@@ -0,0 +1,11 @@
+"""
```

Comment:
Why do you create new exceptions? Can't we simply re raise the existing ones? signing.BadSignature and signing.SignatureExpired are good enough I think.